docs/data/routes/docs/nextjs/creating-components/new-component/en.md

```diff
@@ -0,0 +1,221 @@
+---
+name: new-component
+routeTemplate: ./data/component-templates/article.yml
+title: Walkthrough: Creating a new component
+---
+# Walkthrough: Creating a New Component
+
+This walkthrough demonstrates how to create a new component based on the chosen development workflow: 
+
+* [Using the Sitecore-first development workflow](#using-the-sitecore-first-development-workflow).
+* [Using the code-first development workflow](#using-the-code-first-development-workflow).
+
+## Using the Sitecore-first development workflow
```

Comment:
Probably need to state the assumption here that they have used the `dotnet new` getting started template with a project name `MyProject`

Reply:
I added the note right below the step links.